Source/Dafny/Resolver.cs

```diff
@@ -12389,7 +12402,111 @@ public class ResolveOpts
       return LetPatIn(tok, lhs, rhs, body);
     }
 
+/// <summary>
+/// Desugar monadic binds.
+/// [ var y :- E; Body] ~~> Bind(E, y => Body)
+/// [ :- E; Body] ~~> Bind(E, _ => Body)
+/// Assumes that once resolved, the type of Rhs has a Bind method
+/// </summary>
```

Comment:
Indent these lines

Reply:
I'm not thrilled with the translating the no-LHS `:-` into something that ignores the return value. Dafny doesn't do it that way in other places.